project/MimaSettings.scala

```diff
@@ -28,10 +28,15 @@ object MimaSettings {
       CrossVersion.partialVersion(scalaVersion.value) match {
         case Some((2, 13)) =>
           (previousVersions -- (0 to 4).map("3.3." + _)).map {
-            organization.value % s"${name.value}_${scalaBinaryVersion.value}" % _
+            organization.value %% name.value % _
```

Comment:
https://github.com/lightbend/mima/issues/327

Reply:
This should work and should be the default way of doing things.